sound/soc/intel/common/soc-acpi-intel-tgl-match.c

```diff
@@ -34,12 +75,14 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_machines[] = {
 };
 EXPORT_SYMBOL_GPL(snd_soc_acpi_intel_tgl_machines);
 
+/* this table is used when there is no I2S codec present */
 struct snd_soc_acpi_mach snd_soc_acpi_intel_tgl_sdw_machines[] = {
 	{
-		.link_mask = 0x1, /* this will only enable rt711 for now */
+		.link_mask = 0x3, /* rt711 on link 0 and 2 rt1308s on link 1 */
+		.links = tgl_rvp,
 		.drv_name = "sdw_rt711_rt1308_rt715",
```

Comment:
sdw_rt711_rt1308 ?  link_mask = 0x3.   I will submit a machine driver for this

Reply:
@plbossart What do you think? Should we create a new machine driver for rt711 + 2 rt1308 on the same link?

Reply:
no. we should reuse the same driver and use the mach_params to figure out which dailinks we want to enable, e.g.

rt711
rt711 + I2S rt1308
rt711 + 2x rt1308
rt711 + rt1308 + rt715
rt711 + 2x rt1308 + rt715

Reply:
thanks @plbossart,  I will work on it.

Reply:
@plbossart  for rt711 + I2S rt1308,  machine driver will be first found by acpi id, do we need to support it in this machine driver ?

Reply:
I'd like to avoid multiple machine drivers so yes if we can reuse the same code it's much better.